src/tracer/MARBL_tracers.F90

```diff
@@ -0,0 +1,2353 @@
+!> A tracer package for tracers computed in the MARBL library
+!!
+!! Currently configured for use with marbl0.36.0
+!! https://github.com/marbl-ecosys/MARBL/releases/tag/marbl0.36.0
+!! (clone entire repo into pkg/MARBL)
+module MARBL_tracers
+
+! This file is part of MOM6. See LICENSE.md for the license.
+
+use MOM_coms,            only : EFP_type, root_PE, broadcast
+use MOM_debugging,       only : hchksum
+use MOM_diag_mediator,   only : diag_ctrl
+use MOM_error_handler,   only : is_root_PE, MOM_error, FATAL, WARNING, NOTE
+use MOM_file_parser,     only : get_param, log_param, log_version, param_file_type
+use MOM_forcing_type,    only : forcing
+use MOM_grid,            only : ocean_grid_type
+use MOM_interpolate,     only : external_field, init_external_field, time_interp_external
+use MOM_CVMix_KPP,       only : KPP_NonLocalTransport, KPP_CS
+use MOM_hor_index,       only : hor_index_type
+use MOM_interpolate,     only : forcing_timeseries_dataset
+use MOM_interpolate,     only : forcing_timeseries_set_time_type_vars
+use MOM_interpolate,     only : map_model_time_to_forcing_time
+use MOM_io,              only : file_exists, MOM_read_data, slasher, vardesc, var_desc, query_vardesc
+use MOM_open_boundary,   only : ocean_OBC_type
+use MOM_remapping,       only : reintegrate_column
+use MOM_remapping,       only : remapping_CS, initialize_remapping, remapping_core_h
+use MOM_restart,         only : query_initialized, MOM_restart_CS, register_restart_field
+use MOM_spatial_means,   only : global_mass_int_EFP
+use MOM_sponge,          only : set_up_sponge_field, sponge_CS
+use MOM_time_manager,    only : time_type
+use MOM_tracer_registry, only : register_tracer
+use MOM_tracer_types,    only : tracer_type, tracer_registry_type
+use MOM_tracer_diabatic, only : tracer_vertdiff, applyTracerBoundaryFluxesInOut
+use MOM_tracer_initialization_from_Z, only : MOM_initialize_tracer_from_Z
+use MOM_tracer_Z_init,   only : read_Z_edges
+use MOM_unit_scaling,    only : unit_scale_type
+use MOM_variables,       only : surface, thermo_var_ptrs
+use MOM_verticalGrid,    only : verticalGrid_type
+use MOM_diag_mediator,   only : register_diag_field, post_data!, safe_alloc_ptr
+
+use MARBL_interface,              only : MARBL_interface_class
+use MARBL_interface_public_types, only : marbl_diagnostics_type, marbl_saved_state_type
+
+use coupler_types_mod,      only : coupler_type_set_data, ind_csurf
```

Comment:
Unused? I noticed a few other unused declarations around also.

```suggestion
```

Reply:
Addressed in 78ca691